{{cookiecutter.project_shortname}}/docker-services.yml

```diff
@@ -91,14 +91,19 @@ services:
       - "9300:9300"
   kibana:
     {%- if cookiecutter.elasticsearch == '6'%}
-    image: docker.elastic.co/elasticsearch/kibana-oss:6.6.0
+    image: docker.elastic.co/kibana/kibana:6.6.0
     {%- elif cookiecutter.elasticsearch == '5' %}
     image: kibana:5.6.4
     {%- endif %}
     environment:
+      {%- if cookiecutter.elasticsearch == '6'%}
+      - "ELASTICSEARCH_HOSTS=http://es:9200"
+      {%- elif cookiecutter.elasticsearch == '5' %}
       - "ELASTICSEARCH_URL=http://es:9200"
+      {%- endif %}
       - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
-    ports: ['5601:5601']
+    ports: 
+      - "5601:5601"
```

Comment:
Since the variables for the environment don't overlap you can probably do something like
```
kibana:
  {%- if cookiecutter.elasticsearch == '6'%}
  image: docker.elastic.co/kibana/kibana:6.6.0
  {%- elif cookiecutter.elasticsearch == '5' %}
  image: kibana:5.6.4
  {% endif %}
  environment:
    - "ELASTICSEARCH_HOSTS=http://es:9200"
    - "ELASTICSEARCH_URL=http://es:9200"
    - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
```

Reply:
I just think that it's a bit cleaner for the resulting instance to have only one endpoint variable to avoid confusion in case of modifying it.